mnefun/_mnefun.py

```diff
@@ -401,6 +404,7 @@ def __init__(self, tmin=None, tmax=None, t_adjust=0, bmin=-0.2, bmax=0.0,
         self.must_match = []
         self.on_missing = 'error'  # for epochs
         self.subject_run_indices = None
+        self.plot_pca = False
```

Comment:
Since this doesn't change the existing outputs (just adds some image files), actually we might as well set this to `True` I think. Okay with you?

Last comment I swear :)

Reply:
And leave `params.plot_pca = True` as is in `analysis.py`?

Reply:
Either remove it or add a comment that it's the default but is just there for informative purposes